echopype/calibrate/ek80_complex.py

```diff
@@ -223,20 +282,18 @@ def get_transmit_signal(
     return y_all, y_time_all
 
 
-def compress_pulse(beam: xr.Dataset, chirp: Dict):
+def compress_pulse(backscatter: xr.DataArray, chirp: Dict) -> xr.DataArray:
     """Perform pulse compression on the backscatter data.
 
     Parameters
     ----------
-    beam : xr.Dataset
-        EchoData["Sonar/Beam_group1"] selected with channel subset
+    backscatter : xr.DataArray
+        complex backscatter samples
     chirp : dict
         transmit chirp replica indexed by ``channel``
```

Comment:
If you wanna be thorough, please add Returns here with the type and what it is.

Reply:
Done!